### ISSUE TITLE
RC Spot Check: Molarity 1.5.0-rc.4

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#477]
The rc.3 produced one issue that required a fix and so only spot check is necessary to make sure the fix made it into branch.

[**Simulation**](https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.4/phet/molarity_en_phet.html)

Issues to verify:
 - [x] [Opening links opens two tabs](https://github.com/phetsims/molarity/issues/222)

The fix involved links within the simulation. Please make sure behavior of links found in the about dialog is correct will all forms of input.

Also assigning to @ariel-phet for priority.

Thanks!

**Comment by ariel-phet** [comment]
Current top priority

**Comment by KatieWoe** [comment]
- [ ] Win 10 chrome w/ Jaws
- [ ] Win 10 firefox w/ NVDA
- [ ] Win 10 edge
---
- [ ] Mac 10.15 chrome
- [ ] Mac 10.15 safari w/ VO
---
- [ ] Win 10 IE
- [ ] Mac 10.12 safari
---
- [ ] ChromeOS
- [ ] iPadOS 13 w/ VO

**Comment by KatieWoe** [comment]
Kathryn Woessner:office:  1:17 PM
Quick thing I hadn't noticed about Molarity screen reader before. As you change the sliders it will sometimes say slightly concentrated or very concentrated, but some concentrations don't get such an alert. Is this intended? Can be seen in a11y view

Jesse Greenberg  1:20 PM
Hey, good question - Unfortunately I am only taking it through the publication and I am not familiar with the design. Let me run this question by taliesin.

Jesse Greenberg  1:25 PM
OK I checked with Taliesin, it sounds like that is correct behavior. There are described ranges of concentration and if a change doesn't enter a new range we don't get an alert about it

Kathryn Woessner:office:  1:26 PM
Ok. What are the ranges? I only saw low, very, and max (and pure water)?

Jesse Greenberg  1:26 PM
Hmm, checking code
1:27
Oh man, im a little out of my element here. Im also going to ask MK

Kathryn Woessner:office:  1:28 PM
Gif to help show potential issue:
![lowandveryconcentrated](https://user-images.githubusercontent.com/41024075/74377094-87ff8f00-4da0-11ea-80e6-044a5538144d.gif)


Jesse Greenberg  1:29 PM
Got it, and where is the issue there?

Kathryn Woessner:office:  1:30 PM
It seems hard to tell what concentrations get labels and what they are.

Jesse Greenberg  1:31 PM
Like when "low concentration" and "not very concentrated" comes out?

Kathryn Woessner:office:  1:31 PM
Yeah. Where those ranges are don't seem consistant
1:34
And if you use keyboard nav sometimes they always show up, not just when you cross some threshold
1:34
Sorry we didn't see this

Jesse Greenberg  1:34 PM
OK thanks for walking me through it - sending information to her to make sure all is well.

Kathryn Woessner:office:  1:35 PM
Thanks. Let me know if there needs to be an issue

Kathryn Woessner:office:  1:43 PM
Just made a gif of the behavior when using keyboard nav. Let me know if you want that

Jesse Greenberg  1:53 PM
OK I think i found it - it is very qualitative. It just breaks up the min solute amount and max solute amount into 6 sections. If in a single section change user doesn't enter a new section, we don't get "not very concentrated" (and so on) portion of the alert.

Kathryn Woessner:office:  1:54 PM
Ok. But if you use keyboard nav then sometimes it does keep giving you the alert (but not always)
![alertbehavior](https://user-images.githubusercontent.com/41024075/74377214-b8dfc400-4da0-11ea-8b08-05844ed93dab.gif)

1:54
Is that an issue?

Jesse Greenberg  1:55 PM
No, I don't think so, in fact that may be intended. Ranges may be designed to jump to new region each press. But it ins't perfect.

1:56
This is the behavior TS had in mind and I don't think an issue is necessary unless you have big concerns about it.

Kathryn Woessner:office:  1:56 PM
Ok
1:57
I won’t then. Especially since it’s otherwise close to publishing 

Jesse Greenberg  1:57 PM
Alright, makes sense. Thanks!

Kathryn Woessner:office:  2:03 PM
I'm going to put this convo in the rc issue just as documentation for any future work.
